Aire/ViewModel/PollutantModel.swift

```diff
@@ -49,4 +59,21 @@ class PollutantModel: SCNNode {
         
         return randomVector
     }
+    
+    // Label is visible if node inside patrol distance and is in view of camera
+    func updateLabelVisibility(cameraPos: SCNVector3, inView : Bool ) {
```

Comment:
names with actions (like update) should always _do_ something, in this case, this method is not always updating label visibility. Maybe break down into `isVisible()`and `setLabelVisibility` / manually update `labelVisible`. Or change method name to `updateLabelVisibilityIfNeeded`